pom.xml

```diff
@@ -16,18 +17,18 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <appengine.target.version>1.9.32</appengine.target.version>
+        <appengine.target.version>1.9.34</appengine.target.version>
         <datanucleus.version>3.2.0-m1</datanucleus.version>
     </properties>
 
     <prerequisites>
-        <maven>3.1.0</maven>
+        <maven>3.3.9</maven>
```

Comment:
I am not sure we want this. The plugin is compatible with 3.1.0 and above...
Do you know if we do that, if the plugin would work with 3.1.0.
We've been burnt by the incompatible changes between 3.0.x and 3.1.x